src/atoms/Flex/__tests__/Flex.web.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { renderWithTheme } from '../../../_helpers/testing';
+import View from '../../View';
+import Text from '../../Text';
+import Space from '../../Space';
+import Flex from '../index';
+
+describe('<Flex />', () => {
+  it('renders child by applying flex and space to it', () => {
```

Comment:
this just slipped off my mind sorry.
But the stories are meant to be nested when you have multiple props. Look at this for [reference](https://github.com/razorpay/blade/pull/70/files#diff-2bc383cd6dec7f8a022ce6813e94d55d)

Reply:
Should I nest this inside `it('as')...`

Reply:
add `describe('components composition',() => { it('renders child by applying flex and space properties'`

Reply:
Since all other test cases are having multiple components so should I move all to this `describe` ?

Reply:
no. just for this. because you're testing this explicitly. Rest are focussing on flex properties so that's fine

Reply:
oh, thanks

Reply:
added